tools/downloader/license.txt

```diff
@@ -2,1586 +2,15745 @@ Configuration file for the automation tools includes following models:
 
 ==================================================================================================
 
-* densenet-121, densenet-161, densenet-169, densenet-201 - Densely Connected Convolutional Networks  https://github.com/shicai/DenseNet-Caffe
+* densenet-121 - DenseNet-121 (https://github.com/shicai/DenseNet-Caffe). The following Caffe patch must be used to be able to convert this model: https://github.com/BVLC/caffe/pull/3057/files
 
 License terms:
 
-    Copyright (c) 2016, Zhuang Liu.
-    All rights reserved.
+	Copyright (c) 2016, Zhuang Liu. 
```

Comment:
Could you keep the original 4-space indentation? I think it makes the file more readable, and it would certainly make the diff shorter.